### DISCUSSION TITLE
Emails stuck in pending

### DISCUSSION THREAD

**Discussion by tonydeb42** [postalserver/postal#2440] [Help with using Postal]
all my emails are getting stuck in queue and then never sending , i , tried stopping and restarting server and also creating new credentials. But nothing seems to work. Please help me.

**Comment by willpower232** [comment]
If the emails are not sending, you should check `postal status` to make sure that all parts of Postal are up and running.

**Reply by tonydeb42** [reply]
I ran postal status and it shows me 4 lines I don’t see anything written disabled there also when i started the server everything said working. Im not an expert in this ,should i attach my screenshot ?

**Reply by willpower232** [reply]
It would be good to see a screenshot, I'm not sure how obvious it is these days

**Reply by tonydeb42** [reply]
![Screenshot (3)](https://user-images.githubusercontent.com/122314641/235775019-14d40a55-a9d8-4d0e-8250-03891ab0aef7.png)

Please check this

**Reply by willpower232** [reply]
For some reason you don't have a worker listed, there should be at least 5 rows here.

Have a look at the template docker-compose file.

https://github.com/postalserver/install/blob/main/templates/docker-compose.yml

**Reply by tonydeb42** [reply]
yes from your referenced link I see there is runner and worker missing from mine, how to solve this ? I just normally installed following the instructions and i faced no error during installation.

**Reply by sinnbeck** [reply]
Cant you just update the file, and run `postal stop && postal start` ?

**Reply by tonydeb42** [reply]
i just tried stopping and restarting it and while the process i saw the worker in the list but when i again did postal status it wont show up... Please, any way for this to be solved ?
